airbyte-lib/airbyte_lib/executor.py

```diff
@@ -105,11 +123,27 @@ def _get_installed_version(self):
         venv_name = self._get_venv_name()
         connector_name = self.metadata.name
         return subprocess.check_output(
-            [os.path.join(venv_name, "bin", "python"), "-c", f"from importlib.metadata import version; print(version('{connector_name}'))"],
+            [
+                os.path.join(venv_name, "bin", "python"),
+                "-c",
+                f"from importlib.metadata import version; print(version('{connector_name}'))",
+            ],
```

Comment:
Why are we calling python using subprocess here instead of directly calling `version(<connector-name>)` and handling `importlib.metadata.PackageNotFoundError` ?

Reply:
because it's executed in the separate venv of the connector, not in the python-environment where the airbyte-lib process is running

Reply:
Thanks for the clarification, do you have a doc / diagram to help understand the structure of the environments / projects?